.travis.yml

```diff
@@ -8,11 +8,6 @@ php:
 
 sudo: false
 
-env:
```

Comment:
There hasn't been any need to change that so far, so I think is still ok if we leave it here. Also, it's under version control here so we know when it changes and we know it's not going to get lost (thinking that travis might lose it for whatever crazy reason). So, as part of the changes for this implementation, please revert this change to add the keys back to this file.